config/ruby.yml

```diff
@@ -4,33 +4,59 @@ default: &default
       default: true
       sha256: cca9ddbc958431ff77f61948cb67afa569f01f99c9389d2bbedfa92986c9ef09
       url: https://cache.ruby-lang.org/pub/ruby/3.2/ruby-3.2.0.zip
+      git:
+        tag: v3_2_0
+        branch: ruby_3_2
```

Comment:
I like this! I felt a bit stuck when looking into this because I had a single `git_ref` string might be a tag or might be a branch, and it felt really messy.